syntax = "proto3";

message EachDocData{
    string url = 1;
    string title = 2;
    string date = 3;
    string chars500 = 4;
    string author = 5;
}

message DocData{
    repeated EachDocData eachdoc = 1;
}